packages/bpk-foundations-ios/src/base/aliases.json

```diff
@@ -4,34 +4,18 @@
     "@skyscanner/bpk-foundations-common/app/carousel.json",
     "@skyscanner/bpk-foundations-common/app/spacing.json",
     "@skyscanner/bpk-foundations-common/base/animations.json",
-    "@skyscanner/bpk-foundations-common/base/colors.json"
+    "@skyscanner/bpk-foundations-common/base/colors.json",
+    "@skyscanner/bpk-foundations-common/app/typography.json"
```

Comment:
Nitpick this should come above animations.json so its grouped with other app tokens